### PR TITLE
Update Claims to .NET 6 /3

### LINKED ISSUE #24057: .NET 6: Claims-based authorization
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 25051544-0e4d-1715-95b7-3a8616475127
* Version Independent ID: 50736569-fc04-ec6d-7b33-a3e0c860b1ed
* Content: [Claims-based authorization in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/authorization/claims?view=aspnetcore-6.0)
* Content Source: [aspnetcore/security/authorization/claims.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/authorization/claims.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #24057
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/security/authorization/claims?view=aspnetcore-6.0&branch=pr-en-us-24060#multiple-policy-evaluation) --|--  [Current live version](https://docs.microsoft.com/en-us/aspnet/core/security/authorization/claims?view=aspnetcore-6.0#multiple-policy-evaluation)

### CODE REVIEW

File: aspnetcore/security/authorization/claims.md [hunk 1]
```diff
@@ -22,112 +22,46 @@ An identity can contain multiple claims with multiple values and can contain mul
...
-Claim based authorization checks are declarative - the developer embeds them within their code, against a controller or an action within a controller, specifying claims which the current user must possess, and optionally the value the claim must hold to access the requested resource. Claims requirements are policy based, the developer must build and register a policy expressing the claims requirements.
+Claim based authorization checks:
 
-The simplest type of claim policy looks for the presence of a claim and doesn't check the value.
+* Are declarative.
+* Are applied to Razor Pages, controllers, or actions within a controller. 
+* can ***not*** be applied at the Razor Page handler level, they must be applied to the Page.
 
-Build and register the policy. This takes place as part of the Authorization service configuration, which normally takes part in `ConfigureServices()` in your *Startup.cs* file.
+Claims in code specify claims which the current user must possess, and optionally the value the claim must hold to access the requested resource. Claims requirements are policy based, the developer must build and register a policy expressing the claims requirements.
```
Comment (Rick-Anderson) [inline aspnetcore/security/authorization/claims.md:31]: I had to rewrite this. Long sentences don't MT (Machine Translate) and are difficult for native readers to parse.

**Review by Rick-Anderson — COMMENTED** [review]
* Add Razor Pages
* Update to .NET 6
* Split up long sentence

File: aspnetcore/security/authorization/claims/samples/6.x/WebAll/.gitattributes [hunk 2]
```diff
@@ -0,0 +1,63 @@
+###############################################################################
```
Comment (blowdart) [inline aspnetcore/security/authorization/claims/samples/6.x/WebAll/.gitattributes:1]: This shouldn't be checked in :)

File: aspnetcore/security/authorization/claims/samples/6.x/WebAll/.gitignore [hunk 3]
```diff
@@ -0,0 +1,363 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (blowdart) [inline aspnetcore/security/authorization/claims/samples/6.x/WebAll/.gitignore:1]: I'm even more impressed this made it :)

**Review by blowdart — APPROVED** [review]
Truely impressed you managed to get dot files in the PR :) The text changes look good to me

**Pull request merged**